src/System.Data.SqlClient/tests/FunctionalTests/SqlParameterTest.cs

```diff
@@ -38,5 +38,29 @@ public void ParameterPrecisionOnBaseType()
             Assert.Equal(10, parameter.Precision);
             Assert.Equal(5, parameter.Scale);
         }
+
+        [Fact]
+        public void CreateParameterWithXmlSchema()
```

Comment:
Please add a test with the failing parameter values (3 empty strings, etc.)

      new System.Data.SqlClient.SqlParameter("@Original_Id", System.Data.SqlDbType.BigInt, 0, System.Data.ParameterDirection.Input, 0, 0, "Id", System.Data.DataRowVersion.Original, false, null, "", "", "");

Reply:
Done.